templates/generic/action.html

```
@@ -1,5 +1,5 @@
 {% if action_url %}
-  <a href="{{ action_url }}" class="btn btn-sm join-item {{ button_style|default_if_none:"btn-ghost" }}" {% if disabled %}disabled="disabled"{% endif %} {% if title %}title="{{ title }}"{% endif %} {% if open_url_in_new_tab %}target="_blank" rel="noopener noreferrer"{% endif %}>
+  <a href="{% if not disabled %}{{ action_url }}{% endif %}" class="btn btn-sm join-item {{ button_style|default_if_none:"btn-ghost" }}{% if disabled %} pointer-events-none{% endif %}" {% if disabled %}aria-disabled="true"{% endif %} {% if title %}title="{{ title }}"{% endif %} {% if open_url_in_new_tab %}target="_blank" rel="noopener noreferrer"{% endif %}>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)